src/pybind/rados/rados.pyx

```diff
@@ -3192,6 +3194,43 @@ returned %d, but should return zero on success." % (self.name, ret))
         self.require_ioctx_open()
         return SnapIterator(self)
 
+    def get_id(self):
+        """
+        Get pool id
+
+        :returns: str - pool id
```

Comment:
i don't think the type of the return value is `str`. see https://github.com/ceph/ceph/blob/aaad7dfc0b61c9023fc3295a151448a649173f03/src/include/rados/librados.h#L1003

Reply:
done